### PR TITLE
Update Codeowners

### PR DESCRIPTION
**Pull request by hellwolf**
- adding @0xdavinchee as co-codeowner of /packages/ethereum-contracts/
- adding @kasparkallas to js-sdk too as SDK co-owners.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -11,8 +11,8 @@
 # Packages
```
Comment (0xdavinchee) [inline CODEOWNERS:11]: not sure why but GitHub is giving this error: 
```
Unknown owner on line 4: make sure the team @superfluid-finance/fte exists and has write access to the repository
* @superfluid-finance/fte
```

Comment (0xdavinchee) [reply CODEOWNERS:11]: My guess is that `@superfluid-finance/fte` doesn't have write access?

Comment (hellwolf) [reply CODEOWNERS:11]: right, let me change it [protocol-devs](https://github.com/orgs/superfluid-finance/teams/protocol-devs)

**Review by 0xdavinchee — APPROVED** [review]
LGTM

**Pull request merged**